recipes/orange3-chem/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "Orange3-Chemoinformatics" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "ed7409d2775f94c22e70a9fb86b997bc00fadf9884da4b96ee38e2ab47d45c51" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  skip: True  # [win]
```

Comment:
If you want to skip both you need to use: `[py2k or win]`. However, the tests are passing to I'm wondering if the skip for py2k is needed or if your tests are enough to catch this?

Reply:
I think this actually works for conda-build, it just makes some ways of reading it (maybe the linter?) spit out errors.